docs/reference/filters.md

```diff
@@ -1685,8 +1685,9 @@ authz and access control.
 
 The filter also supports javascript login redirect stub that can be used e.g. to store location hash.
 To enable the stub, add preceding [annotate](#annotate) filter with `oauthGrant.loginRedirectStub` key and
-HTML content that will be served to the client instead of `307 Temporary Redirect` to the authorization URL.
-The filter will replace `{{authCodeURL}}` placeholder in the content with the actual authorization URL.
+content that will be served to the client with `200 OK` status instead of `307 Temporary Redirect` to the authorization URL.
```

Comment:
Was it `200 OK` before this PR as well?

Reply:
yes
https://github.com/zalando/skipper/blob/5a4b32dfb27e3ecd8c72244aa1ef10e4a3a55af4/filters/auth/grant.go#L93-L101